cmd/updatetree/main.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
If this is heavily derived from something pre-existing then use its copyright date?

Reply:
I think I'll leave this as we might well un-derive it when addressing the todos.